dev-docs/modules/jwplayerRtdProvider.md

```diff
@@ -0,0 +1,143 @@
+---
+layout: page_v2
+title: JW Player Real Time Data Provider
+description: makes JW Player's video ad targeting information accessible 
+             to Bid Adapters.
+page_type: module
+module_type: rtd
+module_code : jwplayer
+enable_download : true
+sidebarType : 1
+---
+
+# JW Player RTD Provider
+{:.no_toc}
+
+* TOC
+{:toc}
+
+## Overview
+
+The purpose of this Real Time Data Provider is to allow publishers to target against their JW Player media without 
+being forced to integrate with the Player Bidding product. This prebid module makes JW Player's video ad targeting information accessible 
+to Bid Adapters.
+
+## Implementation for Publishers:
+
+1) Compile the JW Player RTD Provider into your Prebid build:
+
+`gulp build --modules=jwplayerRtdProvider`
+
+2) Publishers must register JW Player as a Real Time Data provider by using `setConfig` to load a Prebid Config containing a `realTimeData.dataProviders` array:
+
+```javascript
+const jwplayerRtdProvider = {
+  name: "jwplayer"
+};
+
+pbjs.setConfig({
+    ...,
+    realTimeData: {
+      auctionDelay: 1000,
```

Comment:
Please lower this example auction delay to more like 100ms. RTD modules are not intended to have auction-length periods.

Reply:
@bretg addressed